frameworks/Crystal/amber/setup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+fw_depends postgresql crystal
+
+shards install --production
+
+crystal build --release --no-debug src/amber.cr
+
+for i in $(seq 1 $((2 * $(nproc --all)))); do
```

Comment:
I'm not sure launching 2 processes per core will gain anything on the actual benchmark machine. I think it'd be best to actually benchmark this on the target machine. Excess process switches are bad.